src/creature.h

```diff
@@ -171,7 +179,7 @@ class Creature : virtual public Thing
 	bool isInvisible() const;
 	ZoneType_t getZone() const
 	{
-		const Tile* tile = getTile();
+		auto tile = getTile();
```

Comment:
Could this be `const auto`
```suggestion
		cont auto tile = getTile();
```